SpaceX/HistoryDetailsScreen/HistoryDetailViewController.swift

```diff
@@ -6,16 +6,206 @@
+    private lazy var infoLabel: UILabel = {
+        let infoLabel = UILabel()
+        infoLabel.numberOfLines = 0
+        infoLabel.font = UIFont(name: "Inter-Regular", size: 15)
+        infoLabel.text = "Info about flight"
+        return infoLabel
+    }()
+    
+    private lazy var infoView: BackroundCardView = {
+        let infoView = BackroundCardView(redLineIsHidden: true)
+        return infoView
+    }()
+    
+    private lazy var videoView: YoutubePlayerView = {
+        let videoView = YoutubePlayerView()
+        return videoView
+    }()
+    
+    private lazy var videoLabel: UILabel = {
+        let videoLabel = UILabel()
+        videoLabel.font = UIFont(name: "Inter-Regular", size: 15)
+        videoLabel.text = "Video"
+        return videoLabel
+    }()
+    
+    private lazy var infoNotFound: UILabel = {
+        let infoNotFound = UILabel()
+        return infoNotFound
+    }()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        setUp()
+        setUpVC()
+        if data?.flight == nil {
+            showInfoNotFound()
+        } else {
+            addSubviews()
+            setUoMissionNameLabel()
+            setUpConstraints()
+            setUpInfoView()
+            createRocketView()
+            setUpVideoView()
+        }
+    }
+    
+    init(data: HomeInfoQuery.Data.History?) {
+        self.data = data
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
     
-    func setUp() {
+    private func setUpVC() {
         view.backgroundColor = .sxWhite
+        navigationItem.title = data?.title
+        navigationController?.navigationBar.prefersLargeTitles = false
+        UILabel.appearance(whenContainedInInstancesOf: [UINavigationBar.self]).adjustsFontSizeToFitWidth = true
+    }
+    
+    private func addSubviews() {
+        view.addSubview(scrollView)
+        scrollView.addSubview(scrollViewContainer)
+        scrollViewContainer.addSubview(missionNameLabel)
+        scrollViewContainer.addSubview(infoLabel)
+        scrollViewContainer.addSubview(infoView)
+        scrollViewContainer.addSubview(videoLabel)
+        scrollViewContainer.addSubview(videoView)
+    }
+    
+    private func setUpConstraints() {
+        scrollView.anchor(top: view.safeAreaLayoutGuide.topAnchor,
+                          leading: view.safeAreaLayoutGuide.leadingAnchor,
+                          bottom: view.safeAreaLayoutGuide.bottomAnchor,
+                          trailing: view.safeAreaLayoutGuide.trailingAnchor)
+        
+        scrollViewContainer.anchor(top: scrollView.topAnchor,
+                                   leading: scrollView.leadingAnchor,
+                                   bottom: scrollView.bottomAnchor,
+                                   trailing: scrollView.trailingAnchor)
+        scrollViewContainer.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
we can remove this line because we did it in extension `anchor`

Reply:
we can remove this line: scrollViewContainer.translatesAutoresizingMaskIntoConstraints = false